index.html

```diff
@@ -0,0 +1,175 @@
+                        <a href="./index.html" class="ticket" onclick="closeMenu()">Get Ticket</a>
+                    </li>
+                </ul>
+            </nav>
+
+            <nav class="desktop-nav">
+                <div class="first-menu-div">
+                    <ul class="first-menu-list">
+                        <li class="list-item">
+                            <a href="#"><i class="fa-brands fa-facebook-f"></i></a>
+                        </li>
+                        <li class="list-item">
+                            <a href="#"><i class="fa-brands fa-twitter"></i></a>
+                        </li>
+                        <li class="list-item">
+                            <a href="#">English</a>
+                        </li>
+                        <li class="list-item">
+                            <a href="#">My Page</a>
+                        </li>
+                        <li class="list-item">
+                            <a href="#">Logout</a>
+                        </li>
+                    </ul>
+                </div>
+                <div class="second-menu-div">
+                    <ul class="second-menu-list">
+                        <li class="menu-item brand">
+                            <a href="./index.html" class="brand">Good<span>Jam</span></a>
+                        </li>
+                        <li class="menu-item">
+                            <a href="./index.html#main-program">Program</a>
+                        </li>
+                        <li class="menu-item">
+                            <a href="./index.html#speakers">Speakers</a>
+                        </li>
+                        <li class="menu-item">
+                            <a href="./about.html">About</a>
+                        </li>
+                        <li class="menu-item get-ticket">
+                            <a href="#">Get Ticket</a>
+                        </li>
+                    </ul>
+                </div>
+            </nav>
+        </header>
+
+        <section class="headline">
+            <div class="menu-icon">
+                <i class="fa-solid fa-bars hamburgur"></i>
+            </div>
+
+            <div class="primary-box">
+                <p class="primary-intro">"Hello! Music Jammers!!"</p>
+                <h1 class="primary-h1">GOOD JAM GLOBAL CONFERENCE <br> 2023</h1>
+            </div>
+            <p class="secondary-text">
+                The long awaited event, Good Jam 2023, is finally here. An always-exciting conference <br class="break-at">where all your curiousity about good music is fulfilled.
+            </p>
+            <div class="date-box">
+                <p class="date">2023.05.14(SUN) ~ 19(FRI)</p>
+                <p class="venue">@ Accra International Center, Ghana</p>
+            </div>
+        </section>
+
+        <section class="main-program" id="main-program">
+            <h3 class="main-program-header section-header">Main Program</h3>
```

Comment:
- [ ] Great work using `hx` for the titles in your HTML files! :+1: However, it's important to ensure that the titles follow a correct hierarchy as per the project requirements. This means that you cannot skip from `h1` to `h3` titles, as this would result in an incorrect hierarchy and poor semantic structure. The correct hierarchy is essential for ensuring that the website is accessible and easy to navigate for all users, including those who use assistive technologies. Therefore, it's important to ensure that the titles in the HTML files are organized in the correct hierarchical order to maintain proper semantics and accessibility. Kindly check your HTML files to fix the hierarchy.

Reply:
Okay, @IngDavidTM. Thanks. I've fixed it now.

Reply:
:+1: